### PR TITLE
add GridAI icon

### PR DESCRIPTION
**Pull request by Borda**
preview3

**Issue:** NA
**Alexa rank:** 191,238
  
https://www.alexa.com/siteinfo/grid.ai

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

adding the https://www.grid.ai/ logo

### CODE REVIEW

File: _data/simple-icons.json
Comment (Borda) [inline _data/simple-icons.json:3537]:
```suggestion
            "title": "gridai",
```

File: _data/simple-icons.json
Comment (Borda) [inline _data/simple-icons.json:3538]:
```suggestion
            "hex": "78FF96",
```

**Comment by PeterShaggyNoble** [comment]
Welcome to Simple Icons, @Borda, and thanks for the contribution 🙂

Unfortunately, your PR has multiple issues that are causing our automated tests to fail - can you have a read through [our contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon) and try to fix up as much as you can based on that an then we can review from there? If you have any problems with anything, feel free to ping one of the team for assistance.

**Comment by Borda** [comment]
@PeterShaggyNoble I am wondering why no check has been triggered?

**Comment by Borda** [comment]
@adamrusted thank you, is there an automatic tooling which would make the centre? finding centre manually (11.938, 12) is quite hard...

**Comment by Borda** [comment]
> you should be able to just select the path in whichever software you're using (Illustrator / Inkscape) and center it horizontally and vertically, fixing the issue

yep sorry for the n00b questions, I am not really a graphic person...
could some misalignment be caused by optimization (postprocessing) svg?

**Comment by adamrusted** [comment]
@Borda no worries! I always forget I've worked with graphics for _years_!
It could be, it depends on what SVGO has to try and fix when optimizing.

**Comment by Borda** [comment]
> You'll need to run `npm install` on your branch before being able to use the linter locally.

that helps, thx :rabbit: 
btw shall I care about hunderts errors `!!! negative-zeros TypeError: iconPath.matchAll is not a function`

**Comment by adamrusted** [comment]
> btw shall I care about hunderts errors `!!! negative-zeros TypeError: iconPath.matchAll is not a function`

**Comment by Borda** [comment]
I have used the Inkscape and then even did some moves, but was not able to get it closer to the centre then  `the center is currently (11.995, 12)`

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:3539]: Before another @simple-icons/maintainers reviews this, it'd be worth updating the source URL to link directly to the file (including blob) that was used.
```suggestion
            "source": "https://github.com/gridai/logos/blob/1e12c83b77abdc22a41566cab232f4db40223895/GridAI-icons/icon-white-48.svg"
```

**Comment by Borda** [comment]
I believe this is ready to review now, Thank you @adamrusted so much for your help!

**Comment by ericcornelissen** [comment]
> > You'll need to run `npm install` on your branch before being able to use the linter locally.
> 
> that helps, thx rabbit
> btw shall I care about hunderts errors `!!! negative-zeros TypeError: iconPath.matchAll is not a function`

Just chiming in her regarding this comment: this is likely because you're running an old, no longer supported, version of NodeJS. This `matchAll` function was added in NodeJS v12, and v12 is the oldest officially supported version of NodeJS. You can verify this by running `node --version`. If you're getting this error while you're over v12 please let us know :smile:

**Comment by PeterShaggyNoble** [comment]
SVG looks perfect to me but should the colour not be black? Also the name is stylised as "Grdi.ai" so the titles in the JSON & SVG will need to be updated and the SVG's filename changed to `griddotai.svg`.

**Comment by Borda** [comment]
> SVG looks perfect to me but should the colour not be black?

the instruction say that SVG shall be monochromatic without any colour, right?

> Also the name is stylised as "Grdi.ai" so the titles in the JSON & SVG will need to be updated and the SVG's filename changed to `griddotai.svg`.

The company is "GridAI" and the web is grid.ai just to have nice name :rabbit:

**Comment by PeterShaggyNoble** [comment]
> the instruction say that SVG shall be monochromatic without any colour, right?

I meant for the `hex` in the JSON data, @Borda.

> The company is "GridAI" and the web is grid.ai just to have nice name

Looking through their website, they consistently refer to themselves as "Grid.ai" throughout, even in legal documents such as their [terms of service](https://www.grid.ai/terms-of-service/) and [privacy policy](https://gridai.wpengine.com/privacy-policy/) so we need to follow suit

**Comment by Borda** [comment]
> > the instruction say that SVG shall be monochromatic without any colour, right?
> 
> I meant for the `hex` in the JSON data, @Borda.

spoke with the Design team in GridAI and they want his green color...

> > The company is "GridAI" and the web is grid.ai just to have nice name
> 
> Looking through their website, they consistently refer to themselves as "Grid.ai" throughout, even in legal documents such as their [terms of service](https://www.grid.ai/terms-of-service/) and [privacy policy](https://gridai.wpengine.com/privacy-policy/) so we need to follow suit

ok, I see, let me include also @williamFalcon @luiscape

**Review by PeterShaggyNoble — APPROVED** [review]
Thanks for your continued work on this one, @Borda, and to @adamrusted for the assistance. Everything looks perfect to me 👍🏻 And, thanks for clarifying the colour choice.

Will hold off on merging until tomorrow to give the Grid.ai team a chance to reply with regard to the name. If they don't get back to us before that but they do want us to run with a different stylisation then we can update that in v5, which is only 2 weeks away.

**Comment by PeterShaggyNoble** [comment]
I see you changed the `title`, @Borda - did they get back to you to confirm that that's the stylisation that they'd prefer?

**Comment by Borda** [comment]
> I see you changed the `title`, @Borda - did they get back to you to confirm that that's the stylisation that they'd prefer?

yes, but I would rather still wait till they final approval/confirmation in this PR

File: _data/simple-icons.json
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:3539]: In the interests of getting this merged for this weekend's release, let's run with "Grid.ai" if we don't hear back from them by the end of today. Like I said, if they would prefer a different stylisation, we can do that in next week's release or, if it constitutes a breaking change, the following week which will be our next major release.

_If_ they do request a different stylisation then we'll need to add "Grid.ai" as an alias for people who will be looking for it using that name.
```suggestion
            "source": "https://github.com/gridai/logos/blob/1e12c83b77abdc22a41566cab232f4db40223895/GridAI-icons/icon-white-48.svg",
            "aliases": {
                "aka": [
                    "Grid.ai"
                ]            
```

Comment (Borda) [reply _data/simple-icons.json:3539]: the alias option  `grid.ai` for original `grid ai` looks as the best for to me

**Comment by PeterShaggyNoble** [comment]
@Borda, _unless_ they say otherwise, we **have** to go with the official stylisation for the title, which is "Grid.ai". The alias should _only_ be added _if_ they request a different stylisation. Can you please update the PR accordingly?

**Comment by PeterShaggyNoble** [comment]
Thanks, @Borda :+1: Will give it 'til the end of the day for feedback from them before merging.

**Comment by williamFalcon** [comment]
this looks great!

**Comment by PeterShaggyNoble** [comment]
Thanks for the feedback, @williamFalcon, much appreciated 🙂

Let's get this merged, so. Thanks for your continued work on this one, @Borda and for the assistance, @adamrusted 👍🏻

**Pull request merged**
